scene/gui/item_list.cpp

```diff
@@ -458,16 +454,13 @@ void ItemList::_input_event(const InputEvent& p_event) {
 					bool selected = !items[j].selected;
 					select(j,false);
 					if (selected)
-						emit_signal("multi_selected",i,true);
+						emit_signal("multi_selected", j, true);
 				}
 			} else {
-
-				if (!mb.doubleclick && !mb.mod.command && select_mode==SELECT_MULTI && items[i].selectable && items[i].selected) {
+				if (!mb.mod.command && select_mode==SELECT_MULTI && items[i].selectable && items[i].selected) {
```

Comment:
You dropped @volzhs's ``!mb.doubleclick``, is that on purpose or a mistake during the conflict resolution?